src/components/ThemeSwitch/ThemeSwitch.tsx

```diff
@@ -0,0 +1,16 @@
+import { Switch } from "@mui/material";
+import { useDispatch } from "react-redux";
+import { toggleTheme } from "../../slices/themeSlice";
+
+const ThemeSwitch = () => {
+
+    const dispatch = useDispatch();
+
+    return (
+        <>
+            <Switch aria-label='switch' onClick={() => dispatch(toggleTheme())}/>
```

Comment:
and maybe make a hook where you return all of the actions from that store
useThemeActions

Reply:
k done